src/pyFAI/load_integrators.py

```diff
@@ -340,3 +340,14 @@
         ocl_sort = None
 else:
     ocl_sort = ocl_azim = ocl_azim_csr = ocl_azim_lut = None
+
+
+# A set of fail-save but precise methods.
+PREFERED_METHODS_1D = IntegrationMethod.select_method(1, split="full", algo="histogram") + \
+                      IntegrationMethod.select_method(1, split="pseudo", algo="histogram") + \
+                      IntegrationMethod.select_method(1, split="bbox", algo="histogram") + \
+                      IntegrationMethod.select_method(1, split="no", algo="histogram")
+PREFERED_METHODS_2D = IntegrationMethod.select_method(2, split="full", algo="histogram") + \
+                      IntegrationMethod.select_method(2, split="pseudo", algo="histogram") + \
+                      IntegrationMethod.select_method(2, split="bbox", algo="histogram") + \
+                      IntegrationMethod.select_method(2, split="no", algo="histogram")
```

Comment:
IntegrationMethod.select_method(1, split="pseudo", algo="histogram")  returns [IntegrationMethod(1d int, bbox split, histogram, cython)] therefore this method is duplicated

```
In [4]: load_integrators.PREFERED_METHODS_1D
Out[4]: 
[IntegrationMethod(1d int, full split, histogram, cython),
 IntegrationMethod(1d int, bbox split, histogram, cython),
 IntegrationMethod(1d int, bbox split, histogram, cython),
 IntegrationMethod(1d int, no split, histogram, python),
 IntegrationMethod(1d int, no split, histogram, cython),
 IntegrationMethod(1d int, no split, histogram, OpenCL, NVIDIA CUDA / NVIDIA RTX A2000 12GB),
 IntegrationMethod(1d int, no split, histogram, OpenCL, Portable Computing Language / pthread-AMD Ryzen 7 PRO 2700 Eight-Core Processor)]


```

Reply:
we just have to write:
```
PREFERED_METHODS_1D = IntegrationMethod.select_method(dim=1, algo="histogram")
PREFERED_METHODS_2D = IntegrationMethod.select_method(dim=2, algo="histogram")
```
right?

Reply:
Since the "pseudo" splitting is deprecated, I agree it should be removed.
But `IntegrationMethod.select_method(dim=1, algo="histogram") ` gives
```
[IntegrationMethod(1d int, no split, histogram, python),
 IntegrationMethod(1d int, no split, histogram, cython),
 IntegrationMethod(1d int, bbox split, histogram, cython),
 IntegrationMethod(1d int, full split, histogram, cython),
 IntegrationMethod(1d int, no split, histogram, OpenCL, NVIDIA CUDA / Quadro M2000),
 IntegrationMethod(1d int, no split, histogram, OpenCL, Portable Computing Language / Quadro M2000),
 IntegrationMethod(1d int, no split, histogram, OpenCL, Portable Computing Language / cpu-Intel(R) Xeon(R) CPU E5-1650 v4 @ 3.60GHz),
 IntegrationMethod(1d int, no split, histogram, OpenCL, NVIDIA CUDA / NVIDIA RTX A5000),
 IntegrationMethod(1d int, no split, histogram, OpenCL, Portable Computing Language / NVIDIA RTX A5000)]
```
thus the default method is both the slowest with the coarsest results, which is not what the default should be.